lib/CGeneralTextHandler.cpp

```diff
@@ -342,6 +342,13 @@ CGeneralTextHandler::CGeneralTextHandler()
 			allTexts.push_back(parser.readString());
 		}
 		while (parser.endLine());
+
+		allTexts[579] = "As you near the %s your body is filled with new energy. You have doubled your normal spell points.";
```

Comment:
Please do not alter `allTexts`, if you need special texts write them in `config\translate.json` and use from there (translate.json is accessible as `localizedTexts` field of `CGeneralTextHandler`).